privacy-on-beam/pbeam/mean_test.go

```diff
@@ -187,6 +187,36 @@ func TestBoundedMeanFloat64FnExtractOutputReturnsNilForSmallPartitions(t *testin
 	}
 }
 
+func TestBoundedMeanFloat64FnWithSpecifiedPartitionsExtractOutputReturnsNilForSmallPartitions(t *testing.T) {
+	for _, tc := range []struct {
+		desc              string
+		inputSize         int
+		datapointsPerUser int
+	}{
+		{"Empty input", 0, 0},
+		{"Input with 1 user with 1 contribution", 1, 1},
+	} {
+
+		fn := newBoundedMeanFloat64Fn(2*1e100, 1e-23, 1, 1, 0, 10, noise.LaplaceNoise, false)
```

Comment:
You don't need to use a delta or multiply epsilon by 2, also for other such tests if you are doing it.

Reply:
Gotcha. Notice how there's no error being returned even though delta is not zero and we are using Laplace noise.. Do you think the check should go in the `BoundedMeanFloat64Fn` rather than in `MeanPerKey`?